android/PhysicalWeb/app/src/main/java/org/physical_web/physicalweb/PhysicalWebBroadcastService.java

```diff
@@ -17,6 +17,9 @@
 package org.physical_web.physicalweb;
 
 import org.physical_web.physicalweb.ble.AdvertiseDataUtils;
+import org.physical_web.collection.PwsClient;
```

Comment:
imports should be in ASCII order, so move these org.physical_web.collection.* imports above org.physical_web.physicalweb.*